hooks/pre-push.d/toml-license-field-test

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
+top=`git rev-parse --show-toplevel`
+status=0
+for file in $(find "$top" -name Cargo.toml); do
+    if [ "$(toml get "$file" package.license)" != "\"Apache-2.0\"" ]; then
+        echo "$file does not include Apache-2.0 in its license field. Please ensure the field is present and includes an Apache license."
```

Comment:
Wrap output.

Reply:
Fixed with two `echo` calls in latest commit revision.